gateway-messages/tests/versioning/v12.rs

```diff
@@ -12,14 +12,21 @@
 //! tests can be removed as we will stop supporting v11.
 
 use super::assert_serialized;
+use gateway_messages::Fwid;
+use gateway_messages::ImageError;
 use gateway_messages::MgsRequest;
+use gateway_messages::RotBootInfo;
+use gateway_messages::RotSlotId;
+use gateway_messages::RotStateV3;
 use gateway_messages::RotWatchdogError;
 use gateway_messages::SerializedSize;
 use gateway_messages::SpComponent;
 use gateway_messages::SpError;
 use gateway_messages::SpResponse;
 use gateway_messages::WatchdogError;
 
+use gateway_messages::SpStateV3;
```

Comment:
With apologies for the messiness of this: v12 has already landed on main (with #216), so these tests need to move to a `v13.rs` module, and the protocol version needs to be bumped to 13.

Reply:
Yes. There was hope that they would land on the same release. I will change that.

Reply:
FWIW, I don't think this is relevant to what release they land in. Since we can install off-release builds, we've been bumping the version in any PR that makes additions to the protocol.

Reply:
done